src/elementary-tweaks/package.yml

```diff
@@ -0,0 +1,21 @@
+name       : elementary-tweaks
```

Comment:
All the elementary packages got prefixed with `pantheon-` instead of `elementary-`, please rename it.